test/integration/utils.ts

```diff
@@ -91,3 +92,7 @@ export async function testTransaction(
   await ledgerAccept(client)
   await verifySubmittedTransaction(client, signedTx as Transaction)
 }
+
+export function getMasterPublicKey(): string {
+  return keypairs.deriveKeypair(masterSecret).publicKey
+}
```

Comment:
This doesn't look like it's used anywhere.

Reply:
Ill remove!